bin/gce-delete-instance

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
 
-uage="Usage: `basename $0` <instance_name> [-q | --quite]"
+delete-instance() {
+  uage="Usage: `basename $0` <instance_name> [-q | --quite]"
 
-instname=$1
-if [ -z "$instname" ]; then
+  instname=$1
```

Comment:
Do you mean to use argument assignment in a function when writing a shell script is not encouraged?